[debug] [2020-10-05T23:45:12.949Z] ----------------------------------------------------------------------
[debug] [2020-10-05T23:45:12.951Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\28573129\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js deploy
[debug] [2020-10-05T23:45:12.951Z] CLI Version:   8.11.2
[debug] [2020-10-05T23:45:12.951Z] Platform:      win32
[debug] [2020-10-05T23:45:12.951Z] Node Version:  v14.11.0
[debug] [2020-10-05T23:45:12.952Z] Time:          Tue Oct 06 2020 01:45:12 GMT+0200 (South Africa Standard Time)
[debug] [2020-10-05T23:45:12.952Z] ----------------------------------------------------------------------
[debug] [2020-10-05T23:45:12.952Z] 
[debug] [2020-10-05T23:45:12.958Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2020-10-05T23:45:12.958Z] > authorizing via signed-in user
[debug] [2020-10-05T23:45:12.958Z] [iam] checking project online-cv-b0c75 for permissions ["firebase.projects.get","firebasehosting.sites.update"]
[debug] [2020-10-05T23:45:12.961Z] > refreshing access token with scopes: ["email","https://www.googleapis.com/auth/cloud-platform","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","openid"]
[debug] [2020-10-05T23:45:12.961Z] >>> HTTP REQUEST POST https://www.googleapis.com/oauth2/v3/token  
 <request body omitted>
[debug] [2020-10-05T23:45:13.483Z] <<< HTTP RESPONSE 200 {"content-type":"application/json; charset=utf-8","vary":"X-Origin, Referer, Origin,Accept-Encoding","date":"Mon, 05 Oct 2020 23:45:13 GMT","server":"scaffolding on HTTPServer2","cache-control":"private","x-xss-protection":"0","x-frame-options":"SAMEORIGIN","x-content-type-options":"nosniff","alt-svc":"h3-Q050=\":443\"; ma=2592000,h3-29=\":443\"; ma=2592000,h3-27=\":443\"; ma=2592000,h3-T051=\":443\"; ma=2592000,h3-T050=\":443\"; ma=2592000,h3-Q046=\":443\"; ma=2592000,h3-Q043=\":443\"; ma=2592000,quic=\":443\"; ma=2592000; v=\"46,43\"","accept-ranges":"none","transfer-encoding":"chunked"}
[debug] [2020-10-05T23:45:13.490Z] >>> HTTP REQUEST POST https://cloudresourcemanager.googleapis.com/v1/projects/online-cv-b0c75:testIamPermissions  
 {"permissions":["firebase.projects.get","firebasehosting.sites.update"]}
[debug] [2020-10-05T23:45:15.514Z] <<< HTTP RESPONSE 200 {"content-type":"application/json; charset=UTF-8","vary":"X-Origin, Referer, Origin,Accept-Encoding","date":"Mon, 05 Oct 2020 23:45:15 GMT","server":"ESF","cache-control":"private","x-xss-protection":"0","x-frame-options":"SAMEORIGIN","x-content-type-options":"nosniff","server-timing":"gfet4t7; dur=1585","alt-svc":"h3-Q050=\":443\"; ma=2592000,h3-29=\":443\"; ma=2592000,h3-27=\":443\"; ma=2592000,h3-T051=\":443\"; ma=2592000,h3-T050=\":443\"; ma=2592000,h3-Q046=\":443\"; ma=2592000,h3-Q043=\":443\"; ma=2592000,quic=\":443\"; ma=2592000; v=\"46,43\"","accept-ranges":"none","transfer-encoding":"chunked"}
[debug] [2020-10-05T23:45:15.516Z] [iam] error while checking permissions, command may fail: FirebaseError: Authorization failed. This account is missing the following required permissions on project online-cv-b0c75:

  firebase.projects.get
  firebasehosting.sites.update
[debug] [2020-10-05T23:45:15.518Z] >>> HTTP REQUEST GET https://firebase.googleapis.com/v1beta1/projects/online-cv-b0c75  
 
[debug] [2020-10-05T23:45:16.158Z] <<< HTTP RESPONSE 403 {"vary":"X-Origin, Referer, Origin,Accept-Encoding","content-type":"application/json; charset=UTF-8","date":"Mon, 05 Oct 2020 23:45:15 GMT","server":"ESF","cache-control":"private","x-xss-protection":"0","x-frame-options":"SAMEORIGIN","x-content-type-options":"nosniff","alt-svc":"h3-Q050=\":443\"; ma=2592000,h3-29=\":443\"; ma=2592000,h3-27=\":443\"; ma=2592000,h3-T051=\":443\"; ma=2592000,h3-T050=\":443\"; ma=2592000,h3-Q046=\":443\"; ma=2592000,h3-Q043=\":443\"; ma=2592000,quic=\":443\"; ma=2592000; v=\"46,43\"","accept-ranges":"none","transfer-encoding":"chunked"}
[debug] [2020-10-05T23:45:16.158Z] <<< HTTP RESPONSE BODY {"error":{"code":403,"message":"The caller does not have permission","status":"PERMISSION_DENIED"}}
[debug] [2020-10-05T23:45:16.161Z] HTTP Error: 403, The caller does not have permission
[debug] [2020-10-05T23:45:16.592Z] FirebaseError: HTTP Error: 403, The caller does not have permission
    at module.exports (C:\Users\28573129\AppData\Roaming\npm\node_modules\firebase-tools\lib\responseToError.js:38:12)
    at Request._callback (C:\Users\28573129\AppData\Roaming\npm\node_modules\firebase-tools\lib\api.js:41:35)
    at Request.self.callback (C:\Users\28573129\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\request\request.js:185:22)
    at Request.emit (events.js:314:20)
    at Request.EventEmitter.emit (domain.js:486:12)
    at Request.<anonymous> (C:\Users\28573129\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\request\request.js:1154:10)
    at Request.emit (events.js:314:20)
    at Request.EventEmitter.emit (domain.js:486:12)
    at IncomingMessage.<anonymous> (C:\Users\28573129\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\request\request.js:1076:12)
    at Object.onceWrapper (events.js:420:28)
    at IncomingMessage.emit (events.js:326:22)
    at IncomingMessage.EventEmitter.emit (domain.js:486:12)
    at endReadableNT (_stream_readable.js:1252:12)
    at processTicksAndRejections (internal/process/task_queues.js:80:21)
[error] 
[error] Error: Failed to get Firebase project online-cv-b0c75. Please make sure the project exists and your account has permission to access it.
